src/module/server-instance/view/server-instance.html

```diff
@@ -17,9 +17,9 @@
                 uib-dropdown-toggle><span translate>ACTIONS</span><span class="caret"></span>
         </button>
         <ul ng-if="ctrl.isDomainMode()" class="dropdown-menu" role="menu" aria-labelledby="dropdownMenu">
-          <li role="presentation" ng-class="ctrl.serverInstance.isRunning()?'disabled':''"><a id-generator="dropdown_menu.start" role="menuitem" ng-click="ctrl.createStartModal()" translate>START</a></li>
-          <li role="presentation" ng-class="!ctrl.serverInstance.isRunning()?'disabled':''"><a id-generator="dropdown_menu.stop" role="menuitem" ng-click="ctrl.createStopModal()" translate>STOP</a></li>
-          <li role="presentation" ng-class="ctrl.serverInstance.isRunning()?'disabled':''"><a id-generator="dropdown_menu.remove" role="menuitem" ng-click="ctrl.createRemoveModal()" translate>REMOVE</a></li>
+          <li role="presentation" ng-class="ctrl.canStart(ctrl.serverInstance)?'':'disabled'"><a id-generator="dropdown_menu.start" role="menuitem" ng-click="ctrl.createStartModal(ctrl.serverInstance)" translate>START_RELOAD_RESTART</a></li>
```

Comment:
Are Reload, Restart and Start mutually exclusive? Or is a state possible where both Reload and Restart are valid?

At the minute, if a user clicks the "Start/Reload/Restart" option they don't know what's actually going to happen behind the scenes. If the actions are mutually exclusive then we should just display the appropriate label (we could re-use `getActionRequired` for this. Otherwise we should display all options as separate menu items and only enable the valid options.